src/app/util/ember-compatibility-functions.cpp

```diff
@@ -415,6 +415,8 @@ CHIP_ERROR GlobalAttributeReader::Read(const ConcreteReadAttributePath & aPath,
         return EncodeCommandList(aPath, aEncoder, &CommandHandlerInterface::EnumerateGeneratedCommands,
                                  mCluster->generatedCommandList);
     default:
+        VerifyOrDieWithMsg(false, DataManagement, "Unexpected global attribute: " ChipLogFormatMEI,
```

Comment:
On first blush, it would appear that the path being passed in could be the path present in the request. In which case, doing a VerifyOrDie would be incorrect.

However, we can only get here if `attributeCluster` is not null in `ReadSingleClusterData`, which can only happen if the path points to a 'valid' global attribute.

Worth putting some kind of comment to that effect here just to assist the reader a bit :)